python/ray/experimental/workflow/workflow_access.py

```diff
@@ -203,6 +214,9 @@ def get_or_create_management_actor() -> "ActorHandle":
         actor = ray.get_actor(MANAGEMENT_ACTOR_NAME)
     except ValueError:
         # the actor does not exist
+        logger.warning("Cannot access workflow manager. It could because "
+                       "the workflow manager exited unexpectedly. A new "
+                       "workflow manager is being created.")
```

Comment:
I feel these two functions are actually the same, right?

Reply:
basically the same, but one gives you a warning and returns the actor, one not.

Reply:
It's ok but just doesn't look nice. Since there are not a lot of lines, maybe it's fine.